src/towncrier/newsfragments/562.removal

```diff
@@ -0,0 +1 @@
+Previously, fragment filenames like ``fix-1.2.3.feature`` identified as closing issue ``3``. Now the identifier of newsfragment is presented as ``fix-1.2.3``.
```

Comment:
Since this "feature" was not documented,
I think that we can consider it a bugfix.
So this filename can be `562.1.bugfix`

I think is best to start with how it works now.

Not sure about ticket vs issue.

Just a suggestion. Feel free to keep your content.

```suggestion
Fragments with filenames like `fix-1.2.3.feature` are now associated with the ticket `fix-1.2.3`.
In previous versions they were incorectly associated to ticket `3`.
```